cliquet/resource.py

```diff
@@ -176,10 +162,10 @@ def raise_304_if_not_modified(self, record=None):
 
             if current_timestamp <= modified_since:
                 response = HTTPNotModified()
-                self.add_timestamp_header(response)
+                self._add_timestamp_header(response)
                 raise response
 
-    def raise_412_if_modified(self, record=None):
+    def _raise_412_if_modified(self, record=None):
```

Comment:
Should this stay public, in case a library user would like to add it to other endpoints?